docs/data_driven_testing.adoc

```diff
@@ -232,6 +245,26 @@ to other data variables:
 include::{sourcedir}/datadriven/DataSpec.groovy[tag=sql-data-variable-assignment]
 ----
 
+== Multi-Variable Assignment
+
+Like with data pipes, you can also assign to multiple variables in one expression, if you have some object Groovy
+can iterate over. Unlike with data pipes, the syntax here is identical to standard Groovy multi-assignment syntax:
```

Comment:
We could link to https://groovy-lang.org/semantics.html#_multiple_assignment

Reply:
I'm not sure it is a good idea.
The only additional information there is irrelevant and at most confusing, as it is about declaring the variables you assign by having `def` before the tuple and optionally having types at the individual variables, which is not valid here.
If you want me to add it, I'll add it, I'm just not sure about it.
wdym?